sandbox-templates/mdx-loader-next/pages/index.mdx

```diff
@@ -0,0 +1,8 @@
+# heading
+
+* list
+* item
+
+```js
+function () {}
+```
```

Comment:
Would be good to showcase not just markdown, but also JSX, ESM, and expressions. How about:

```suggestion
export const planet = 'World'

# Hello, {planet}!

<div style={{backgroundColor: 'tomato', padding: '8px', borderRadius: '3px'}}>
  * list
  * items
</div>
```

Reply:
updated